modules/infinite-scroll/themes/twentyten.php

```diff
@@ -8,43 +8,48 @@
 /**
  * Add theme support for infinity scroll
  */
-function twenty_ten_infinite_scroll_init() {
+function jetpack_twentyten_infinite_scroll_init() {
 	add_theme_support( 'infinite-scroll', array(
-		'container' => 'content',
-		'render'    => 'twenty_ten_infinite_scroll_render',
-		'footer'    => 'wrapper',
+		'container'      => 'content',
+		'render'         => 'jetpack_twentyten_infinite_scroll_render',
+		'footer'         => 'wrapper',
+		'footer_widgets' => jetpack_twentyten_has_footer_widgets(),
```

Comment:
This can handle an array of sidebar IDs, any reason we don't just do that instead of calculating it ourself (as we do sometimes when we need more complex checks involving whether we're mobile)?

https://github.com/Automattic/jetpack/blob/a695ee8419d43a8d0423acee5c954fc5f39a919a/modules/infinite-scroll/infinity.php#L180-L192